### ISSUE TITLE
Status of external subtitle files not correctly reported

### ISSUE THREAD

**Issue by fongd** [JakeWharton/plex-orphaned-files#2]
`.srt` files are incorrectly reported as not indexed by Plex even though they may be. If there's a way to determine whether Plex is using these, then this should be fixed. Otherwise, perhaps a command-line option to ignore external subtitle files would be more convenient than a generic "exclude" option as proposed in #1?

**Comment by JakeWharton** [comment]
The info on rules are here: https://support.plex.tv/articles/200471133-adding-local-subtitles-to-your-media/

It's a shame they aren't indexed like media files. Or maybe they are and it's simply in another array.

In either case, I'll take a look soon. Thanks for the report.

**Comment by fongd** [comment]
Just to confirm, the `.srt` files are properly named and appear in Plex in the Subtitles dropdown for the corresponding episodes so they are recognized by Plex.

**Comment by JakeWharton** [comment]
Yeah sorry that link was for my own reference when I go to try and fix this.

**Comment by jonfairbanks** [comment]
I am also seeing the `srt` bug along with a few .nfo and .jpg files. 

Is the intention of this tool to list missing media only or files of any type that Plex is not watching?

**Comment by JakeWharton** [comment]
Unless I can query the extensions with the API, I don't want to use an opt-in list of extensions and would prefer that you opt-out. For jpg and nfo you'll be able to use #1.

**Comment by JakeWharton** [comment]
Despite being indexed internally, the file path of the subtitles is not included in the API responses...

That's unfortunate. It means we have to try to mirror Plex's behavior here to mark a file as being used.
